src/runtime/internal/dom.ts

```diff
@@ -63,23 +72,23 @@ export function listen(node: EventTarget, event: string, handler: EventListenerO
 }
 
 export function prevent_default(fn) {
-	return function(event) {
+	return function (event) {
```

Comment:
this formatting change is unnecessary. same for the rest of the changes below this

Reply:
uups.. I must have forgotten to disable auto-formatting. I fixed it